src/sys/wait.rs

```diff
@@ -1,58 +1,105 @@
-use libc::{self, c_int};
+//! This module contains the `wait()` and `waitpid()` functions, which are used to wait on and
+//! obtain status information from child processes. These provide more granular control over
+//! child management than the primitives provided by Rust's standard library, and are critical
+//! in the creation of shells and managing jobs on *nix platforms.
+//!
+//! # Example
```

Comment:
This should be `# Examples` in line with the [special sections](https://doc.rust-lang.org/stable/book/first-edition/documentation.html#special-sections) syntax